pyproject.toml

```diff
@@ -65,17 +65,19 @@ filterwarnings = [
   'ignore:\n            Sentinel is not a public part of the traitlets API:DeprecationWarning',
 ]
 
-[tool.bandit]
-# Excluding tests because bandit doesn't like `assert`.
-exclude_dirs = ["docs/conf.py", "tests"]
+[tool.ruff.lint]
+select = ["I", "UP", "B", "S"]
```

Comment:
Still less than in scipp. In particular, I would like to use `T20`, `PGH`, `RUF` in order to enforce proper use of `noqa` and `type: ignore` directives. (Among some other nice checks.)

But more importantly, this deselects `F` which is the default flake8-based lints.

Reply:
Yup, I am playing around with some more rules to try to get a generalised one.

Also for some reason ruff think scipp is pandas and tries to run https://docs.astral.sh/ruff/rules/#pandas-vet-pd on the codebase (enforcing `.values` to `.to_numpy`)

Reply:
> Also for some reason ruff think scipp is pandas and tries to run https://docs.astral.sh/ruff/rules/#pandas-vet-pd on the codebase (enforcing .values to .to_numpy)

This may be an example of a checker that cannot safely be used in all codebases. As I understand it, ruff simply inspects the local parts of the AST but does not do any type checking. So it cannot know whether it is dealing with a pandas data frame or something else.